package/kernel/mac80211/patches/subsys/310-mac80211-split-mesh-fast-tx-cache-into-local-proxied.patch

```diff
@@ -175,12 +175,12 @@ Signed-off-by: Felix Fietkau <nbd@nbd.name>
 +	ether_addr_copy(key.addr, addr);
  	cache = &sdata->u.mesh.tx_cache;
  	spin_lock_bh(&cache->walk_lock);
--	entry = rhashtable_lookup(&cache->rht, addr, fast_tx_rht_params);
+-	entry = rhashtable_lookup_fast(&cache->rht, addr, fast_tx_rht_params);
 -	if (entry)
 -		mesh_fast_tx_entry_free(cache, entry);
 +	for (i = MESH_FAST_TX_TYPE_LOCAL; i < MESH_FAST_TX_TYPE_FORWARDED; i++) {
 +		key.type = i;
-+		entry = rhashtable_lookup(&cache->rht, &key, fast_tx_rht_params);
++		entry = rhashtable_lookup_fast(&cache->rht, &key, fast_tx_rht_params);
```

Comment:
@nbd168 I need your input here as I moved it to `rhashtable_lookup_fast` and I am not sure if that is appropriate

Reply:
LGTM

Reply:
I am unsure, that is why I asked Felix as he is the original author.
But I think it may be OK if I am understanding the upstream commit: https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/net/mac80211/mesh_pathtbl.c?h=v6.6.15&id=2c1d8b046866b653d55bd50e088335ee34760122

Reply:
@nbd168 looking at the code, rcu is locked with [mpp_path_add](https://elixir.bootlin.com/linux/latest/C/ident/mpp_path_add) but not for every [__mesh_path_del](https://elixir.bootlin.com/linux/latest/C/ident/__mesh_path_del)

Worth investigating maybe? 

Anyway this is ot for this change and is just packing a change upstream. Sorry for the noise